include/core/jmessage.h

```diff
@@ -53,7 +53,8 @@ enum JMessageType
 	J_MESSAGE_DB_INSERT,
 	J_MESSAGE_DB_UPDATE,
 	J_MESSAGE_DB_DELETE,
-	J_MESSAGE_DB_QUERY
+	J_MESSAGE_DB_QUERY,
+	J_MESSAGE_RESET
```

Comment:
`J_MESSAGE_DB_RESET`?

Reply:
the intention was to have one reset function for all backend types - that is why there is no DB prefix